### PR TITLE
Automatic port of System.Runtime.Intrinsics documentation

### PR DESCRIPTION
**Pull request by carlossanlop**
Ported using [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool).

Since these are highly specialized APIs, I expect we don't need a detailed review in the summaries containing the intrinsic signature. So please just help reviewing the natural language descriptions, like class summaries.

### CODE REVIEW

**Comment by kunalspathak** [comment]
If I compare x86 intrinsics methods, it has extra `<para>` inside `<summary>`. May be that renders better.

https://github.com/dotnet/dotnet-api-docs/blob/a43f1387573cd0cf3e82e1fcc7a0e7938b89d70a/xml/System.Runtime.Intrinsics.X86/Avx2.xml#L127

E.g. in your changes, i see ` <summary>float64x2_t vabsq_f64 (float64x2_t a) A64: FABS Vd.2D, Vn.2D</summary>` which would end up rendering in a single line. Might be good to have it in different line.

File: xml/System.Runtime.Intrinsics.X86/Avx.xml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="Windows-1252"?>
 <Type Name="Avx" FullName="System.Runtime.Intrinsics.X86.Avx">
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.X86/Avx.xml:2]:
```suggestion
<Type Name="Avx" FullName="System.Runtime.Intrinsics.X86.Avx">
```

File: xml/System.Runtime.Intrinsics.X86/Sse2.xml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="Windows-1252"?>
 <Type Name="Sse2" FullName="System.Runtime.Intrinsics.X86.Sse2">
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.X86/Sse2.xml:2]:
```suggestion
<Type Name="Sse2" FullName="System.Runtime.Intrinsics.X86.Sse2">
```

File: xml/System.Runtime.Intrinsics.Arm/Sha256.xml [hunk 3]
```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class provides access to the ARM SHA256 hardware instructions via intrinsics</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.Arm/Sha256.xml:23]:
```suggestion
    <summary>This class provides access to the ARM SHA256 hardware instructions via intrinsics.</summary>
```

File: xml/System.Runtime.Intrinsics.Arm/Sha1.xml [hunk 4]
```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class provides access to the ARM SHA1 hardware instructions via intrinsics</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.Arm/Sha1.xml:23]:
```suggestion
    <summary>This class provides access to the ARM SHA1 hardware instructions via intrinsics.</summary>
```

File: xml/System.Runtime.Intrinsics.Arm/Crc32.xml [hunk 5]
```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class provides access to the ARM Crc32 hardware instructions via intrinsics</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.Arm/Crc32.xml:23]:
```suggestion
    <summary>This class provides access to the ARM Crc32 hardware instructions via intrinsics.</summary>
```

File: xml/System.Runtime.Intrinsics.Arm/ArmBase.xml [hunk 6]
```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class provides access to the ARM base hardware instructions via intrinsics</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.Arm/ArmBase.xml:23]:
```suggestion
    <summary>This class provides access to the ARM base hardware instructions via intrinsics.</summary>
```

File: xml/System.Runtime.Intrinsics.Arm/Aes.xml [hunk 7]
```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class provides access to the ARM AES hardware instructions via intrinsics</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.Arm/Aes.xml:23]:
```suggestion
    <summary>This class provides access to the ARM AES hardware instructions via intrinsics.</summary>
```

File: xml/System.Runtime.Intrinsics.Arm/AdvSimd.xml [hunk 8]
```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class provides access to the ARM AdvSIMD hardware instructions via intrinsics</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Intrinsics.Arm/AdvSimd.xml:23]:
```suggestion
    <summary>This class provides access to the ARM AdvSIMD hardware instructions via intrinsics.</summary>
```

**Comment by carlossanlop** [comment]
@kunalspathak @echesakovMSFT @tannergooding can you please take a look again?

File: xml/System.Runtime.Intrinsics.Arm/AdvSimd.xml [hunk 9]
```diff
@@ -44,7 +44,10 @@
       </Parameters>
       <Docs>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>
+          <para>int16x8_t vabsq_s16 (int16x8_t a) </para>
+          <para>A32: VABS.S16 Qd, Qm A64: ABS Vd.8H, Vn.8H</para>
```
Comment (kunalspathak) [inline xml/System.Runtime.Intrinsics.Arm/AdvSimd.xml:49]: should the `A64` be under separate `<para>`?

Comment (kunalspathak) [reply xml/System.Runtime.Intrinsics.Arm/AdvSimd.xml:49]: I see it that way for `Aes.xml`.

Comment (carlossanlop) [reply xml/System.Runtime.Intrinsics.Arm/AdvSimd.xml:49]: Yes. Good catch, I missed this file. Fixed.

**Comment by carlossanlop** [comment]
@dotnet/docs can I get a sign-off so I can merge?

**Pull request merged**